packages/modal/package.json

```diff
@@ -12,12 +12,14 @@
   "publishConfig": {
     "access": "public"
   },
+  "peerDependencies": {
+    "react-transition-group": "^4.2.1"
```

Comment:
Might have to add this to the root package.json so it's still installed on first installations. If you remove node_modules and re-install does this still work?